api/src/clojure/org/akvo/flow_api/unilog/unilog.clj

```diff
@@ -77,13 +85,19 @@
                                                   (comp form-deleted :form-id)
                                                   (remove
                                                     (comp form-instance-deleted :id)
-                                                    (distinct (keep ::form-instance-changed final)))))]
+                                                    (distinct (keep ::form-instance-changed final)))))
+               data-point-deleted (set (keep ::data-point-deleted final))
+               data-point-changed (set (remove
+                                         (comp data-point-deleted #(Long/parseLong %) :id)
+                                         (keep ::data-point-changed final)))]
            {::unilog-id (:id (last final))
             ::form-instance-deleted form-instance-deleted
             ::form-updated form-updated
             ::form-deleted form-deleted
             :forms-to-load (apply conj form-updated (keys form-instances-grouped-by-form))
-            ::forms-instances-grouped-by-form form-instances-grouped-by-form}))
+            ::forms-instances-grouped-by-form form-instances-grouped-by-form
+            ::data-point-changed data-point-changed
```

Comment:
Remember that Datapoints need to be permission. Maybe we plan to add it later, but as with the form-instance data we could build some of the logic already. 

Up to you if you want to do it with the permissions work.